ipyopenlayers/example.py

```diff
@@ -21,13 +21,23 @@ class TileLayer(Widget):
     _view_name = Unicode('TileLayerView').tag(sync=True)
     _view_module = Unicode(module_name).tag(sync=True)
     _view_module_version = Unicode(module_version).tag(sync=True)
+    url = Unicode('https://{a-c}.tile.openstreetmap.org/{z}/{x}/{y}.png').tag(sync=True)
 
-    url = Unicode().tag(sync=True)
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+        self.observe(self._on_url_change, 'url')
 
+    def _on_url_change(self, change):
+        if 'new' in change:
+            new_url = change['new']
+            print('1')
+            self.send({'event': 'url_changed', 'new_url': new_url})
```

Comment:
```suggestion
```

I believe we don't need this, we can react to the `url` trait change in the front-end directly without sending a manual message